### PR TITLE
Footer styling fixes (spacing, float, size, etc...)

### PR DESCRIPTION
**Pull request by caesar2164**

### CODE REVIEW

File: lagunita/lms/static/sass/partials/lms/theme/custom_overrides/_footer.scss [hunk 1]
```diff
@@ -1,5 +1,11 @@
+.main-container {
+    padding-bottom: 0;
```
Comment (stvstnfrd) [inline lagunita/lms/static/sass/partials/lms/theme/custom_overrides/_footer.scss]: How does this interact w/ other pages that don't have (or it's not as apparent) this issue?

Comment (caesar2164) [reply lagunita/lms/static/sass/partials/lms/theme/custom_overrides/_footer.scss]: It's a small change, so it doesn't change the spacing much, but I'd be cool waiting for the Upstream bootstrap PR first?

Comment (stvstnfrd) [reply lagunita/lms/static/sass/partials/lms/theme/custom_overrides/_footer.scss]: Ping now that the bootstrap PR has landed..

**Comment by stvstnfrd** [comment]
cherry-picked.

**Pull request closed without merging**